### DISCUSSION TITLE
Provide issue-specific URL within projects

### DISCUSSION THREAD

**Discussion by tiagoboldt** [community/community#29871] [Projects and Issues]
While using Github Projects, I would like to be able to click on an issue and share the current URL with others, so that they can access the issue directly within the project. As of today, the URL when interacting with a project remains static, despite whatever interaction one is making with the project.

**Comment by robopzet** [comment]
I also miss this to link to related issues of a project in a description.

**Comment by joelibaceta** [comment]
As far as I know, it is possible to access a specific issue through an URL 

github.com/`ORGANIZATION-OR-USER`/`REPOSITORY`/issues/`ISSUE_NUMBER`

🤔 I think is easily shareable too from issues tab -> right-click in any issue > copy link address

**Reply by robopzet** [reply]
As I understand this issue is not about the issues in a repository, they are easy to share.  I thought here it's about issues in a project. At least that is what I'm looking for.

**Reply by joelibaceta** [reply]
![Captura de pantalla 2022-09-08 a la(s) 16 15 42](https://user-images.githubusercontent.com/864790/189059040-82fb57a1-c985-41ec-9b9f-610b4649f4ec.png)

For project issues, we have these options in the detailed view 🤔. So `copy link` should solve this, at least for non-drafts issues.

**Reply by tiagoboldt** [reply]
That copies the link to the issue's own page, not to the project with the issue open on the side panel.

**Comment by MPLew-is** [comment]
This would also be very useful for draft issues which exist solely on the project board and have no other link whatsoever.

**Reply by robopzet** [reply]
The label **Draft** is a bit weird. You can assign the issues to people, and I've used a custom property to keep track of their status. Some of them are never converted to "real" issues, but do get finished.

**Comment by mattcosta7** [comment]
https://github.blog/changelog/2023-01-05-github-issues-january-5th-update/

We recently shipped urls that link directly to the projects sidepanel open to an issue or draft issue
